setup.py

```diff
@@ -11,6 +11,7 @@
     'tornado>=4.2.1',
     'cryptography>=0.2.1',
     'pyopenssl',
+    'PyYAML'
```

Comment:
Specified minimum of 3.11, max of 4 for the current time in case of breaking API changes.
!resolved